homework/unique_ptr/UniquePointer.hpp

```diff
@@ -50,6 +50,10 @@ class UniquePointer {
         return *get();
     }
 
+    pointer operator->() const noexcept{
+	   return get();
+	} 
+
```

Comment:
I think as we are within class namespace we can access the pointer directly instead of using `get()`.

Reply:
By using 'get' internally, we can trap access to pointer while debugging in one place. Its also name of what to do and hides how its done so it may be done different way by changing only 'get' method.